### PR TITLE
feat: Drops filmStripOnly mode.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: css/filmstrip/_vertical_filmstrip.scss [hunk 1]
```diff
@@ -152,11 +152,6 @@
     .filmstrip {
```
Comment (damencho) [inline css/filmstrip/_vertical_filmstrip.scss:152]: For some unknown for me reason, if I delete this `.filmstrip-only` then the local thumbnail is added at the bottom of the page 🤷‍♂️

**Comment by damencho** [comment]
https://github.com/jitsi/jitsi-meet-torture/pull/317

File: css/filmstrip/_vertical_filmstrip.scss (hunk 1 above)
Comment (damencho) [reply css/filmstrip/_vertical_filmstrip.scss]: Thanks @hristoterezov for the css lesson :) 🤦

**Review by hristoterezov — APPROVED** [review]

File: css/filmstrip/_vertical_filmstrip.scss (hunk 1 above)
Comment (hristoterezov) [reply css/filmstrip/_vertical_filmstrip.scss]: np :)

**Review by saghul — APPROVED** [review]
I love the smell of deleted code.

**Comment by martinellimarco** [comment]
The iFrame API still mention filmStripOnly [here](https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-iframe)
I was following the example there and got confused because it was not having any effect, then I've found this merge.

**Comment by saghul** [comment]
Oops, updated the docs, cheers!

**Comment by bmatzner** [comment]
Is there a discussion somewhere that lead to the removal of this feature? We were looking to implement just that: a film-strip only mode. Is there a preferred alternative way to achieve that?

**Comment by damencho** [comment]
What was mentioned is tileview with one column when we were discussing this internally.

**Comment by jamesjessian** [comment]
This is also a surprise for us. We are using the filmstripOnly mode, so we can't upgrade at the moment. Is there a discussion somewhere for migrating to the new version? i.e. what we need to change for people who were using the API with filmstripOnly mode? I've only managed to find this commit so far. We'll search for the tileview/column options in the meantime.

**Pull request merged**